Sources/Misc/DangerousSettings.swift

```diff
@@ -124,6 +127,8 @@ internal protocol InternalDangerousSettingsType: Sendable {
     /// Whether `HTTPClient` will fake server errors
     var forceServerErrors: Bool { get }
 
+    var forcedServerErrors: [String: [NetworkError]]? { get }
```

Comment:
I wonder with this if we should obsolete the `forceServerErrors` property instead, since you could potentially do the same thing with this I guess... but maybe it's ok to leave as is for now.